packages/api/src/command/medical/document/document-query.ts

```diff
@@ -89,6 +90,15 @@ export async function queryDocumentsAcrossHIEs({
     },
   });
 
+  analytics({
+    event: EventTypes.documentQuery,
+    distinctId: cxId,
+    properties: {
+      requestId,
+      patientId,
+    },
+  });
```

Comment:
What's this for? So we can calculate the total across all HIEs?

Reply:
The reason for this one and the one about you commented on that only have properties `requestId` and `patientId` is so we can calculate 
-  num DQs with success WH sent
-  num CQs with success WH sent

We dont have the requestId when calling the endpoint so i add this one as the point of reference